maintainers/maintainer-list.nix

```diff
@@ -14689,6 +14689,12 @@
     githubId = 1437166;
     name = "Xia Bin";
   };
+  sochotnicky = {
+    email = "stanislav+github@ochotnicky.com";
+    github = "sochotnicky";
+    githubId = 55726;
+    name = "Stanislav Ochotnický";
+  };
```

Comment:
This should be it's own commit before the tremotesf: init and called `maintainers: add sochotnicky`